src/providers/BackgroundBlurProvider/docs/BackgroundBlurProvider.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { BackgroundBlurProvider } from '../';
+
+<Meta title="SDK Providers/BackgroundBlurProvider" />
+
+# BackgroundBlurProvider
+
+The `BackgroundBlurProvider` provides a function transforming a normal video device to a `DefaultVideoTransformDevice, and also whether or not the Background Blur feature is supported. 
+Background Blur related logs can be found in the browser developer tools when the `BackgroundBlurProvider` is used within the app component tree.
+
+This provider is independent from `MeetingProvider`. You can put `BackgroundBlurProvider` wherever you want in the component tree. The `BackgroundBlurProvider` loads the required worker assets
+when mounted, which may impact performance.
+
+## State
+
+```typescript
+{
+  // Whether Background Blur is supported. The default value is undefined, then changes to true or false.
+  isBackgroundBlurSupported: boolean | undefined;
+
+  // A function to transform a video input device to a DefaultVideoTransformDevice.
+  createBackgroundBlurDevice: (device: Device) => Promise<DefaultVideoTransformDevice>;
+}
+
+```
+
+You should see either "processor is supported" or "processor is not supported" in your browser developer tools based on whether 
```

Comment:
[Can do as a follow-up]: Link to JS SDK guide documentation here on the support if available.